app/src/test/java/com/aragh/kotlin2/screen/useralbums/UserAlbumsTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.aragh.kotlin2.screen.useralbums
+
+import com.aragh.kotlin2.interactor.UserAlbums
+import com.aragh.kotlin2.api.UserAlbumsApi
+import com.aragh.kotlin2.data.Album
+import com.aragh.kotlin2.data.NewAlbum
+import kotlinx.coroutines.experimental.Unconfined
+import okhttp3.Headers
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito.*
+import retrofit2.Response
+import java.util.concurrent.CompletableFuture
+
+
+class UserAlbumsTest {
+
+  val viewMock = mock(ViewContract::class.java)
+  val userAlbumsApi = mock(UserAlbumsApi::class.java)
+  val userAlbums = UserAlbums(userAlbumsApi)
+  val presenter = UserAlbumsPresenter(userAlbums, Unconfined)
+
+
+  @Before
+  fun setupViewer() {
+    presenter.view = viewMock
+  }
+
+  @Test
+  fun albumsAreShown() {
+    `when`(userAlbumsApi.userAlbums(1)).thenReturn(
```

Comment:
Ooor you could import when as https://www.youtube.com/watch?v=weRHyjj34ZE

Reply:
i like when better, backticks don't bother me :P